src/core/client/stream/tabs/Live/LiveComment/LiveCommentContainer.css

```diff
@@ -57,6 +98,19 @@
   height: 20px;
 }
 
+.parentBodyHighlight {
+  width: 200px;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  text-align: left;
+
+  color: var(--palette-text-000);
+
+  margin-right: var(--spacing-1);
+  height: 20px;
```

Comment:
Don't use `height: 20px` here on this typography element. Did you mean `line-height` instead?